testing/chain.go

```diff
@@ -51,7 +49,7 @@ type SenderAccount struct {
 // is used for delivering transactions through the application state.
 // NOTE: the actual application uses an empty chain-id for ease of testing.
 type TestChain struct {
-	t *testing.T
```

Comment:
had to make this public in order to support the custom test chain from a new constructor

Reply:
Can you point me to the issue? Not sure I follow

Reply:
This line:

https://github.com/tharsis/evmos/pull/282/files#diff-e1b6b9bd60ea81ed06c40d66a72dfba81bb74c77f3fee03c99197cbd1249f083R16

Reply:
Fair enough